src/corona_hakab_model/consts.py

```diff
@@ -245,26 +254,62 @@ class ContagiousStochasticState(ContagiousState, StochasticState):
         class ImmuneTerminalState(ImmuneState, TerminalState):
             pass
 
+        # In order to deal with the latent + pre-symptomatic probabilities, we split latent state in two:
+        # latent-presymp - which will have the duration of the incubation period (minus 1 day)
+        # Presymp will have a short distribution of 1-3 days
+        # latent-asymp - will have the durations of latent, summed by each day
+        # probability for each is same as probability from latent to presymp and asymp
+
         susceptible = SusceptibleTerminalState("Susceptible", test_willingness=self.susceptible_test_willingness)
         latent = ImmuneStochasticState("Latent", detectable=False, test_willingness=self.latent_test_willingness)
-        silent = ContagiousStochasticState(
-            "Silent", contagiousness=self.silent_infection_ratio, test_willingness=self.silent_test_willingness
+        latent_presymp = ImmuneStochasticState(
+            "Latent-Presymp",
+            detectable=False,
```

Comment:
That's one of the problems with this simplification - you should be detectable in presymp, and undetectable in latent, and here you had to choose only one detectability option for this mesh-up state.

Maybe we should ask someone whether we should make this state detectable?

Reply:
Good point, I am talking to Amit Spinrad today so I'll ask him

Reply:
I'm resolving this comment. If needed, the constant will be changed as a hotfix.